thruk.conf

```diff
@@ -977,4 +978,4 @@ cookie_auth_direct_agents         = (wget|curl|python|Go-http-client)
     #hostmenu    = ^Host            # matches all hosts starting with 'Host'
     #demomenu    = ^Demo;           # matches all services on the 'Demo' host
     #demomenu    = ^Test;           # a menu name can be used multiple times
-</action_menu_apply>
+</action_menu_apply>i
```

Comment:
there is a trailing "i"

Reply:
OK to everything; I'll see what I can do. Thanks

Reply:
I made and uploaded the changes this morning. Let me know what you think.